bin/ak-entrypoint-light.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
I see no benefit for this. It will not work with `docker exec` for instance. So having to repeat the twice the args in the docker-compose is still needed.